App/views.py

```diff
@@ -1,15 +1,22 @@
 from django.http import HttpResponse
 from django.shortcuts import render
+import math
 import json
 import requests
 import toolforge
 
 def main2(request):
     if(request.method=='POST'):
+        flag = 1
```

Comment:
This works, but if you want a nicer approach, you can wrap everything in a try...catch and throw an exception when the user is not found etc.

Reply:
Done.